client/jetpack-cloud/sections/partner-portal/payment-methods/stored-payment-method-api.ts

```diff
@@ -31,33 +37,6 @@ export async function saveCreditCard( {
 	return response;
 }
 
-interface UpdateCreditCard {
-	token: string;
-	stripeConfiguration: StripeConfiguration;
-	useAsPrimaryPaymentMethod: boolean;
-}
-
-export async function updateCreditCard( {
```

Comment:
Note: This is unused (and would never have worked anyway since you cannot update a purchase without a purchase id).